helix-term/src/commands.rs

```diff
@@ -216,6 +217,27 @@ pub fn move_line_start(cx: &mut Context) {
     doc.set_selection(view.id, selection);
 }
 
+pub fn move_first_nonwhitespace(cx: &mut Context) {
+    let (view, doc) = cx.current();
+
+    let selection = doc.selection(view.id).transform(|range| {
+        let text = doc.text();
+        let line_idx = text.char_to_line(range.head);
+
+        let max_to = doc.text().len_chars().saturating_sub(1);
```

Comment:
`to`/`max_to` is unnecessary for these two functions, you calculate it but it's not used anywhere

Reply:
That's right, I forgot to clean up. Done.